### PR TITLE
Added prop autocomplete

### PR DESCRIPTION
**Pull request by mariusrak**
I've added autocomplete property for TextFields into standard templates.

### CODE REVIEW

File: packages/uniforms-antd/src/TextField.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function Text(props: TextFieldProps) {
   return wrapField(
     props,
     <Input
+      autoComplete={props.autoComplete}
```
Comment (radekmie) [inline packages/uniforms-antd/src/TextField.tsx:17]: Is this one actually needed? It should not as `{...filterDOMProps(props)}` below does not remove `autoComplete`. Same goes for `uniforms-material` theme.

Comment (mariusrak) [reply packages/uniforms-antd/src/TextField.tsx:17]: I wanted to use it with _-unstyled_ but it was assigned to `<div>` instead of `<input>`

Comment (radekmie) [reply packages/uniforms-antd/src/TextField.tsx:17]: Yeah, I see that both `uniforms-semantic` and `uniforms-unstyled` required this change, hence I'm asking about `uniforms-antd` and `uniforms-material`. Also, what about `uniforms-bootstrap3` and `uniforms-bootstrap4`? Could you add tests there as well?

Comment (mariusrak) [reply packages/uniforms-antd/src/TextField.tsx:17]: I'm not sure about _-antd_ and _-material_, but I've expected same behavior from them as _-unstyled_. I've missed the bootstrap versions, so I'll look at them also.

Comment (mariusrak) [reply packages/uniforms-antd/src/TextField.tsx]: So, in new commits, tests are added and if it is not needed to explicitly set the value it is not there.

File: packages/uniforms-bootstrap4/src/TextField.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ function Text(props: TextFieldProps) {
       className={classnames(props.inputClassName, 'form-control', {
         'is-invalid': props.error,
       })}
+      autoComplete={props.autoComplete}
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/TextField.tsx:21]: I've just checked and while both Bootstrap themes correctly understand `autoComplete` now, it also appears on the `div` rendered by `wrapField`. To fix that, do add `autoComplete` to the list of omitted values above.

**Review by radekmie — CHANGES_REQUESTED** [review]

File: packages/uniforms-bootstrap4/src/TextField.tsx (hunk 2 above)
Comment (mariusrak) [reply packages/uniforms-bootstrap4/src/TextField.tsx:21]: Fixed and added tests for that, hopefully it's okay now.

**Review by radekmie — APPROVED** [review]

**Pull request merged**